Makefile

```diff
@@ -1,18 +1,23 @@
 .PHONY: start
 
 # Read values as needed from .env
+#
+# Optionally pass 'environment=<something>' to load env vars from '.env-something' instead
+# Useful to separate i.e. dev from testing databases
+#
 # If using the same variables in recipes that need to use a dotenv file other
 # than .env, remember to check that no values from .env are being used
 # inadvertently.
-ENVFILE := $(if $(environment), .env-test-e2e, .env)
+ENVFILE := $(if $(environment), .env-$(environment), .env)
+ifneq (,$(wildcard $(ENVFILE)))
+    include $(ENVFILE)
+    export
+endif
```

Comment:
I like this, but I think that piggybacking on the existing `environment` make params may break CI and/or running some things (tests) in Docker locally. It should be ok to refactor the existing use of this param to make it possible to use it both to read an environment-specific .env file and to differentiate the test runs between CI and local Docker Compose setups.

Reply:
we briefly discussed this during a call, but for reference:
Right now, the 'environment' argument is used more like a feature flag of sorts, and not so much as a way to define different environments. This change pushes to actually restore it to being basically an identifier between different sets of configurations. If we need to further tune specific behavior, we can add env vars to said environments, to control behavior. However, I don't think that will be necessary, at least for now